test-e2e/web/README.md

```diff
@@ -0,0 +1,53 @@
+# Simple Web App with Wasm
+
+## Preparations
+
+Follow these steps only if there are changes in the `mopro-wasm` package. Otherwise, these steps are not required.
+
+### Compile Halo2 Circuits to Wasm
+
+This web test page requires three Wasm circuits: `halo2-plonk-fibonacci`, `halo2-hyperplonk-fibonacci`, and `halo2-gemini-fibonacci`.  
+Refer to [`mopro-wasm/README.md`](../../mopro-wasm/README.md) for more details.
+
+Use `wasm-pack` to generate halo2 wasm packages for this testing web app with the following commands:
+
+```bash
+mopro-wasm $ wasm-pack build --target web --out-dir ../test-e2e/web/halo2-plonk-fibonacci -- --features plonk
+mopro-wasm $ wasm-pack build --target web --out-dir ../test-e2e/web/halo2-hyperplonk-fibonacci -- --features hyperplonk
+mopro-wasm $ wasm-pack build --target web --out-dir ../test-e2e/web/halo2-gemini-fibonacci -- --features gemini
```

Comment:
I assumed that the user would eventually use only one type of backend—Plonk, HyperPlonk, or Gemini in Halo2—even for testing purposes. However, if you think it's better to have a single Wasm package in `test-e2e/web`, such as `test-e2e/web/halo2-wasm-circuit`, I can modify it to follow your suggestion

Reply:
or maybe even not specify `halo2`?
I hope it can be more general for applications
so like `test-e2e/web/mopro` or `test-e2e/web/mopro-pkg`
to let users know the package is generated by mopro and in the documentation we can guide them to use
```ts
import * as mopro_wasm from './mopro/mopro_wasm.js';
```
to import and use functions